src/hotspot/share/runtime/vmThread.cpp

```diff
@@ -271,14 +272,14 @@ static void post_vm_operation_event(EventExecuteVMOperation* event, VM_Operation
 
 void VMThread::evaluate_operation(VM_Operation* op) {
   ResourceMark rm;
-
   {
     PerfTraceTime vm_op_timer(perf_accumulated_vm_operation_time());
     HOTSPOT_VMOPS_BEGIN(
                      (char *) op->name(), strlen(op->name()),
```

Comment:
@dholmes-ora given that I'm editing `VMThread::evaluate_operation` and your comments about code style - do you want me to also change this type cast to `(char*)` to align with the preferred style?

Reply:
Please do. Thanks